.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  spm:
+    name: Build and Test on Xcode 16
+    runs-on: macos-14
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+      - name: Select Xcode Version
+        run: sudo xcode-select --switch /Applications/Xcode_16.app/Contents/Developer
+      - name: Build and Test Framework
+        run: |
+          xcrun swift test -c release --enable-code-coverage -Xswiftc -enable-testing
```

Comment:
I don't have permissions to set up Codecov in this repo so I didn't set that up. We’ll Get There™

Reply:
https://github.com/fdiaz/SafeDI-Formatter/issues/6